octomap_msgs/Octomap octomap
---
geometry_msgs/Point[] up_facing_points
